### ISSUE TITLE
Question about running eval mode 

### ISSUE THREAD

**Issue by SgtVincent** [ShunChengWu/3DSSG#8]
Hi! 

I have a doubt about pipeline to run the model in eval mode. 

I followed your instructions to generate ground truth data for training/evaluation. In `./data/gen_data_gt.py`, you provide 3  choices of splits: `['train', 'test', 'validation']`. I run the scripts for those 3 split types. The generated `test_scans.txt` `relationships_test.json` files are EMPTY, that I wonder if this is the expected behavior. 

Besides, when I run main.py  in eval mode, the argument 'split_type' is set to 'test_scans' which is empty and causes an exception. https://github.com/ShunChengWu/3DSSG/blob/24d4ac520c3441fb6815d0d4890dd845e10e5cc7/src/SceneGraphFusionNetwork.py#L50

How can I fix this error? Do I need to generate test data in another way or should I change the 'split_type' to 'validation_scans'?

**Comment by ShunChengWu** [comment]
Hi,

since I have no right to access the real test set of 3RScan (only the author of the dataset should have the right for it). The evaluation set we used in this work is actually the validation set. 
What we did is split the training set into two splits, with any percentage you want, typically 0.8, 0.2 or 0.9, 0.1. Then change the name accordingly. Then use the validation set as the test set. You can pass a list of scan_ids to our data generation script by `--target_scan  /path/to/scan_ids.txt`.
